Sources/Async/File/FileReader.swift

```diff
@@ -5,8 +5,7 @@ import Foundation
 public protocol FileReader {
     /// Reads the file at the supplied path
     /// Supply a queue to complete the future on.
-    func read<S>(at path: String, into stream: S, chunkSize: Int)
-        where S: Async.InputStream, S.Input == UnsafeBufferPointer<UInt8>
+    func read(at path: String, chunkSize: Int) -> AnyOutputStream<UnsafeBufferPointer<UInt8>>
```

Comment:
What's wrong with providing the _into_ input stream? That allows us to keep the strong type info and makes implement a file reader simpler.